codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -236,7 +236,8 @@ private void writePendingBytes() {
 
     /**
      * This will allocate bytes by stream weight and priority for the entire tree rooted at {@code parent}, but does
-     * not write any bytes.
+     * not write any bytes. Priority is generally distributed proportionally to siblings in the priority tree, however
+     * we ensure that at least 1 byte is assigned to each stream in order to make progress.
```

Comment:
@Scottmitch I'm unsure where you've left this ... are you ok with it as-is? Any thoughts on @buchgr's comment above?